Tribler/community/allchannel/community.py

```diff
@@ -218,7 +217,7 @@ def create_channelcast(self):
                 # we've send something to this address, add to blocklist
                 self._blocklist[candidate] = now
 
-                nr_torrents = sum(len(torrent) for torrent in torrents.values())
+                nr_torrents = sum(len(infohash) for infohash in torrents.itervalues())
```

Comment:
This is incorrect, the torrents dict acutally contains a list of infohashes per community identifier.
If you want to improve upon the previous version, rename torrent to community_torrents.